pkg/auth/authorizer/interfaces.go

```diff
@@ -56,12 +59,20 @@ func (f AuthorizerFunc) Authorize(a Attributes) error {
 	return f(a)
 }
 
+type MetaResource int
+
+const (
+	ApiVersions MetaResource = 1 << iota
+	HealthCheck
+)
+
 // AttributesRecord implements Attributes interface.
 type AttributesRecord struct {
-	User      user.Info
-	ReadOnly  bool
-	Namespace string
-	Resource  string
+	User      		user.Info
+	ReadOnly  		bool
+	Namespace 		string
+	Resource  		string
+	MetaResource 	MetaResource
```

Comment:
In OpenShift, we settled on `NonResourcePath` (the path for the request) and `Verb`.  This allowed us to express things like, "you can do GETs on `/healthz` and `/healthz/*`".

Enumerating every available endpoint is a losing battle once we you want to subdivide things.  For example, what if we decided to subdivide health to support individual component health.  It also doesn't really buy much control since anyone wanting to write such a policy it has to know exactly they want anyway.